test/testcafe/spec/DeviceChallengePollViewFailure_spec.js

```diff
@@ -34,32 +35,73 @@ const mock = RequestMock()
-  await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().innerText).eql('Cancel and take me to sign in');
-  await t.expect(deviceChallengePollPageObject.getFooterSignOutLink().length).eql(0);
-  await t.expect(logger.count(
-    record => record.response.statusCode === 200 &&
+test
+  .requestHooks(logger, mock)('probing and polling APIs are sent and responded', async t => {
+    const deviceChallengePollPageObject = await setup(t);
+    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Verifying your identity');
+    await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().innerText).eql('Cancel and take me to sign in');
+    await t.expect(deviceChallengePollPageObject.getFooterSignOutLink().length).eql(0);
+    await t.expect(logger.count(
+      record => record.response.statusCode === 200 &&
       record.request.method !== 'options' &&
       record.request.url.match(/introspect|2000/)
-  )).eql(3);
-  await t.expect(logger.count(
-    record => record.response.statusCode === 200 &&
+    )).eql(3);
+    await t.expect(logger.count(
+      record => record.response.statusCode === 200 &&
       record.request.url.match(/challenge/) &&
       record.request.body.match(/challengeRequest":"eyJraWQiOiI1/)
-  )).eql(1);
-  await t.expect(deviceChallengePollPageObject.form.getErrorBoxText()).contains('You do not have permission to perform the requested action');
-  await t.expect(deviceChallengePollPageObject.getSpinner().getStyleProperty('display')).eql('none');
-  await t.expect(deviceChallengePollPageObject.getFooterSignOutLink().innerText).eql('Take me to sign in');
-  await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().length).eql(0);
-});
+    )).eql(1);
+    await t.expect(deviceChallengePollPageObject.form.getErrorBoxText()).eql('You do not have permission to perform the requested action');
+    await t.expect(deviceChallengePollPageObject.getSpinner().getStyleProperty('display')).eql('none');
+    await t.expect(deviceChallengePollPageObject.getFooterSignOutLink().innerText).eql('Take me to sign in');
+    await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().length).eql(0);
+  });
+
+test
+  .requestHooks(logger, deviceInvalidatedErrorMsg)('add title when device or account is invalidated', async t => {
+    const deviceChallengePollPageObject = await setup(t);
+    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Verifying your identity');
+    await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().innerText).eql('Cancel and take me to sign in');
+    await t.expect(deviceChallengePollPageObject.getFooterSignOutLink().length).eql(0);
+    await t.expect(deviceChallengePollPageObject.form.getErrorBoxText()).eql(
+      'Couldn’t verify your identity\n\n' + errorDeviceInvalid.messages.value[0].message);
+    await t.expect(deviceChallengePollPageObject.getSpinner().getStyleProperty('display')).eql('none');
```

Comment:
IMO you don't have to repeat what has been already verified by other tests.. Keep this test to strictly check for what you added..